### ISSUE TITLE
Problems with trying to run the program in evaluate mode

### ISSUE THREAD

**Issue by MrMixit** [wvangansbeke/Sparse-Depth-Completion#31]
Hi!

I am trying to load your pretrained model into the main.py script and running with `--evaluate` but I am having problems with getting everything running. I got the files needed from KITTI, and used your `download_raw_files.sh`, but modified it to only download validation data into `Data/val`. The model you have trained is saved in a folder I created in `Saved`, but I don't know how to send this path to the program. Also, when trying to run I have to supply a `--mod` which I don't know how to format, and I get an error:
```
main.py: error: argument --mod: invalid choice: '/home/emdkaai/Desktop/Sparse-Depth-Completion/Saved/pretrained/model_best_epoch.pth' (choose from 'mod')
```
Could you give me any suggestions on what to try?

Best,
Adam

**Comment by wvangansbeke** [comment]
Hi @MrMixit,

Thank you for trying out the code.

You can indeed use the `--evaluate` flag to compute the results on the validation set. If you look at [this line](https://github.com/wvangansbeke/Sparse-Depth-Completion/blob/master/main.py#L203), you can see it looks for the best_model* in the `saved` directory. So you need to pass the folder where you saved your best model by specifying the `--save_path` flag. (So `Saved` in your case). You shouldn't change `--mod` , because it is used to change the architecture.

Hope this helps.

**Comment by MrMixit** [comment]
Thank you for the fast response! This was of great help, I managed to get it running now.

Thanks,
Adam
